lrth06/view-counter
# add color customization support

# Description


Add support to take in additional query parameter `text=<HEX CODE>`  to allow for customization of text color.

## Relevant Issues



## How Has This Been Tested?




### Testing Process


Verified in local environment that query is being respected

## Verification steps


Deploy locally and visit `localhost:3000?text=000` to verify text has changed from default white, to black color.
